index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="css/style.css" />
+  <title>My Portfolio</title>
+</head>
+
+<body>
+  <div class="container">
+    <header class="navbar">
+      <a href="#">Dan Logo</a>
+      <span> <img src="images/burger.png" alt="toggle icon" /> </span>
+    </header>
+    <section class="header">
+      <div class="header-title">
+        <h1>Hey there. I'm Dan Elifeleti</h1>
+        <h2 class="green">I'm a software developer</h2>
```

Comment:
- [ ] kindly make your headings bolder if font-weight didn't do it automatically

Reply:
ok